.circleci/config.yml

```diff
@@ -331,6 +331,31 @@ jobs:
             docker push "tendermint/tendermint"
             docker logout
 
+  reproducible_builds:
+    <<: *defaults
```

Comment:
This image is set to `circleci/golang`. Since this updates automatically, you might have issues in the future. The image should be locked down to a version.

Reply:
It does not really affect this build step. No Go compilation takes place here, everything happens in a docker container that is created and run by `./scripts/gitian-build.sh`